ui/src/helpers/api.js

```diff
@@ -76,4 +76,12 @@ export function questionsQuery() {
   return request
     .get('/server/questions')
     .set('Content-Type', 'application/json');
+}
+
+export function saveQuestions(record) {
```

Comment:
Should `record` here be `questions`, based on how the server interprets the request body?